js/device-detection.js

```diff
@@ -1,27 +1,3 @@
-/**
- Original Code
-
-function getMobileOperatingSystem() {
-    var userAgent = navigator.userAgent || navigator.vendor || window.opera;
-
-    // Windows Phone must come first because its UA also contains "Android"
-    if (/windows phone/i.test(userAgent)) {
-        return "Windows Phone";
-    }
-
-    if (/android/i.test(userAgent)) {
-        return "Android";
-    }
-
-    // iOS detection from: http://stackoverflow.com/a/9039885/177710
-    if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
-        return "iOS";
-    }
-
-    return "unknown";
-}
-
- */
```

Comment:
Why did you remove this? It looks like this should work

Reply:
It should but halfway through doing this I decided it this was a great opportunity to kick the tires on Chat GPT's coding ability. This didn't connect to anything so it was just taking up space.